CMakeLists.txt

```diff
@@ -146,7 +146,7 @@ endif()
 
 if (OCOS_ENABLE_BERT_TOKENIZER)
   # Bert
-  file(GLOB bert_TARGET_SRC "operators/tokenizer/wordpiece*.*")
+  file(GLOB bert_TARGET_SRC "operators/tokenizer/wordpiece*.*" "operators/tokenizer/basic_tokenizer.*" "operators/tokenizer/bert_tokenizer.*")
```

Comment:
Hi, what's the difference between new bert tokenizer and existing wordpiece tokenizer? Should they both be built using one flag?

Reply:
BertTokenizer consist of BasicTokenizer and WordPieceTokenizer, here I will split them.